v2.2/managed-frequently-asked-questions.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Frequently Asked Questions
+summary:
+toc: true
+build_for: [managed]
+---
+
+## Cluster basics
+
+### Is my cluster secure?
+
+Yes. We create individual sub-accounts and VPCs for each cluster within the cloud provider. These VPCs are firewalled from each other and any other outside connection, unless whitelisted for SQL and Web UI ports. The whitelist is an additional layer of protection for your cluster. Connections will only be accepted if they come from a whitelisted IP address, which protects against both compromised passwords and any potential bugs in the server.
+
+We use separate certificate authorities for each cluster, and all connections to the cluster over the internet use TLS 1.2.
+
+### Is my cluster isolated? Does it share resources with any other clusters?
+
+Managed CockroachDB is a single-tenant offering and resources are not shared between clusters.
+
+### How do I connect to my cluster?
+
+We will provide you with a connection string for each region (or one region if its a single region cluster) as well as a global load balancer, a CA certificate, and your first database username and password. You can use this information to connect to your cluster.
+
+### What machines do you run your cluster on?
+
+Our machines run with 4 CPUs, 8 CPUs, or 16 CPUs per node.
+
+## Cluster maintenance
+
+### How do I change the configurations on my cluster?
+
+Today, you can reach out to [Support]([Support](https://support.cockroachlabs.com/hc/en-us) for most cluster changes. In early 2019, you will be able to make changes using a self-service management console.
+
+### How do I add nodes?
+
+To add nodes, you can reach out to [Support]([Support](https://support.cockroachlabs.com/hc/en-us) to request to add nodes. Our team will work with you to update your cluster configurations. We expect this to be self-service next year.
```

Comment:
"To add nodes,... to add nodes." > "To add nodes..." 

[Support]([Support](https://support.cockroachlabs.com/hc/en-us) > [Support](https://support.cockroachlabs.com/hc/en-us)